### PR TITLE
Update readme with citation information

### PR DESCRIPTION
**Pull request by guillemcortes**

### CODE REVIEW

**Comment by guillemcortes** [comment]
Ok, tests are passing now. Li fas una ullada? @genisplaja

**Comment by genisplaja** [comment]
hey @guillemcortes, I can see it's still trying to run the 3.10 python test for windows and macOS. I guess the reason is because this PR was created before merfing the other? But the 3.11 tests are being run here. Just to make sure we are not forgetting something!

File: CITATION.cff
Comment (genisplaja) [inline CITATION.cff:1]: What is this version exactly? The version of soundata? Don't think so, right?

File: README.md
Comment (genisplaja) [inline README.md]: Maybe we could ask them to cite the paper instad of the _version of soundata they are using_?

**Review by genisplaja — CHANGES_REQUESTED** [review]
hey! Couple of minor questions/suggestions but looks good! Thanks for this one as well @guillemcortes :)

File: CITATION.cff
Comment (guillemcortes) [reply CITATION.cff:1]: the .cff file was autogenerated by JOSS. It is the latest version of CITATION.cff https://github.com/citation-file-format/citation-file-format/blob/main/CHANGELOG.md#120---2021-05-31

File: README.md
Comment (guillemcortes) [reply README.md]: hmm I guess it is important that people says which version of Soundata they have run. They could cite the Zenodo entry, for instance but I guess we can reword it to something like:

> When working with datasets, please indicate the version of `soundata` that you are using **AND** include the reference of the dataset, which can be found in the respective dataset loader using the `cite()` method.

Since we are already telling everyone to cite the paper by redirecting them to CITATION.cff file. Is this better?

File: CITATION.cff
Comment (genisplaja) [reply CITATION.cff:1]: right

File: README.md
Comment (genisplaja) [reply README.md]: mmmm maybe it'd be cool to make a bit more clear when and how to cite soundata. I guess there are two cases:
* People cite soundata as an idea I guess, because they do something similar, or they get inspired for another library.... I suppose that in this case, they should cite the paper
* People use soundata in their pipeline. Maybe in this case we should recommend to cite the zenodo entry?

**Comment by guillemcortes** [comment]
> hey @guillemcortes, I can see it's still trying to run the 3.10 python test for windows and macOS. I guess the reason is because this PR was created before merfing the other? But the 3.11 tests are being run here. Just to make sure we are not forgetting something!

yes, I think it is because I created this PR before merging the drop py3.7 and update to python3.11 PR was merged, so now this branch is up to date with main, but it still expects the py3.10 jobs to run. It is not a problem, they will "disappear" once we merge this as they dissapeared when we merged #182

File: README.md
Comment (genisplaja) [inline README.md]: this **AND** should change, right? They should cite the software with the DOI if using soundata in the pipeline, and citing the paper if they use it as inspiration for something similar, right?

**Review by genisplaja — CHANGES_REQUESTED** [review]

**Comment by genisplaja** [comment]
@guillemcortes what do you think?

**Comment by guillemcortes** [comment]
What about now @genisplaja ?

**Review by genisplaja — APPROVED** [review]
looks good :)

**Pull request merged**
